.env.example

```diff
@@ -1,2 +1,4 @@
 export MADELINE_EMAIL_FROM=madeline@theworkingworld.org
 export MADELINE_HOSTNAME=madeline.theworkingworld.org
+export QB_OAUTH_CONSUMER_KEY=
+export QB_OAUTH_CONSUMER_SECRET=
```

Comment:
Another thing just occurred to me -- have we added anything to the Readme for QB stuff? Should probably be something about where to get keys, and what you can and can't do with out adding a key.